src/CMakeLists.txt

```diff
@@ -409,7 +409,7 @@ configure_file(i18n/translations.qrc "${CMAKE_CURRENT_BINARY_DIR}" COPYONLY)
 set(SOURCES ${SOURCES} ${CMAKE_CURRENT_BINARY_DIR}/translations.qrc ${QM_FILES})
 
 # Create an rpi-imager_lupdate target
-qt_add_lupdate(TS_FILES ${TRANSLATIONS} SOURCE_TARGETS ${PROJECT_NAME})
+qt_add_lupdate(TS_FILES ${TRANSLATIONS} SOURCE_TARGETS ${PROJECT_NAME} OPTIONS -no-obsolete)
```

Comment:
This should probably be done in a separate PR, as I suspect there's multiple `.ts` files containing "vanished" translations.

Reply:
If you make the cngahges in cmakelist,.txt at next build the cmakelist.txt will purge all .ts languages fro obsolete strings.

Reply:
To be clear I'm not disagreeing with this change; IMHO it'd just be cleaner if it was done in a separate PR from one that's also updating Italian translation strings.

Reply:
(but of course it's up to @tdewey-rpi to have the final say!)

Reply:
OK. Please take care that there is also an update about instalelr script.
What's the best steps for you?
Close this PR and open a new PR just for Italian languiag update?
Then wait that someone merge this PR about Italian language update and after that create a new PR for Cmakelis.txt?
Then wait that someone merge this PR about cmkaelist.txt and create a new PR for instalelr script?

Reply:
Please have a look at https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches

The "typical approach" is to PR separate features from different branches, rather than making all your changes on the same branch.